docs/program/sdks/_index.md

```diff
@@ -254,6 +300,11 @@ go run ~/myCode/myViamFile.py
 
 For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+For an example, see [this execution demo.](https://github.com/viamrobotics/viam-typescript-sdk/tree/main/examples/vanilla)
```

Comment:
This looks like a typescript link. Probably the best link for C++ would be https://github.com/viamrobotics/viam-cpp-sdk/tree/main/src/viam/examples/dial, though we don't have a great README for it at this time.

Reply:
I think it actually should be https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md again. We're looking for something that points to runnign the code